web/themes/custom/fmcg_material/sass/styles.scss

```diff
@@ -2273,3 +2098,100 @@ form#views-exposed-form-job-board-page {
     border-radius: 2px;
   }
 }
+
+@media (min-width: 1256px) {
+  .navbar-hamburger-menu {
+    display: none !important;
+  }
+}
+
+.path-frontpage div#main-content, .path-user div#main-content, .path-user-login div#main-content {
+  background: none;
+}
+
+.password-strength__title, .password-confirm {
+  font-family: Roboto;
+  font-size: 13px;
+  color: #DBE4EB;
+}
+
+.password-strength__meter {
+  height: 0.75em;
+  margin-top: 0.5em;
+  background-color: #211F21;
+}
+
+.password-strength__indicator {
+  width: 0;
+  height: 100%;
+  background-color: #FDD22B;
+}
+
+.job-post-organization-form {
+  .form-item label {
+    font-size: 16px;
+    font-weight: 400;
+    font-family: Poppins;
+    line-height: 2em;
+    color: #211F21;
+  }
+}
+
+.section-wrapper .section {
+  max-width: 800px;
+  margin: 0 auto;
+  background: #FFFFFF;
+  padding: 0 20px 75px 20px;
+  position: relative;
+}
+
+.job-post-organization-form .form-item label {
+  font-size: 16px;
+  font-weight: 400;
+  font-family: Poppins;
+  line-height: 2em;
+  color: #211F21;
+}
+
+.form-item input[type=text] {
+  display: block;
+  width: 100%;
+  max-width: 550px;
+  padding-left: 10px;
+  padding-right: 10px;
+  background-color: #FFFFFF;
+  float: right;
+  margin-bottom: 20px;
+}
+
+.fieldset-legend {
+  font-size: 20px;
+  font-weight: 600;
+  background-color: #DBE4EB;
+  margin-right: 12px;
+}
+
+.js-form-item .form-item .js-form-wrapper .form-wrapper {
```

Comment:
I'm not sure this will ever apply? As far as i know, .js-form-item and .form-item always go together.